src/sparseml/optim/helpers.py

```diff
@@ -198,7 +198,9 @@ def parse_recipe_variables(
 def update_recipe_variables(recipe_yaml_str: str, variables: Dict[str, Any]) -> str:
     """
     :param recipe_yaml_str: YAML string of a SparseML recipe
-    :param variables: variables dictionary to update recipe top level variables with
+    :param variables: variables dictionary to update recipe top level variables with.
+                      If recipe contains stages, it will parse the whole recipe
```

Comment:
Comments should be tab spaced on new lines

Reply:
I should learn this once for all haha, thanks